modules/theme-tools/site-breadcrumbs.php

```diff
@@ -21,16 +28,18 @@ function jetpack_breadcrumbs() {
 	}
 
 	$breadcrumb = '';
+	$position   = 1;
 
 	if ( $is_post_type_hierarchical ) {
 		$post_id   = get_queried_object_id();
 		$ancestors = array_reverse( get_post_ancestors( $post_id ) );
 		if ( $ancestors ) {
 			foreach ( $ancestors as $ancestor ) {
-				$breadcrumb .= '<span itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a href="' . esc_url( get_permalink( $ancestor ) ) . '" itemprop="item"><span itemprop="name">' . esc_html( get_the_title( $ancestor ) ) . '</span></a></span>';
+				$breadcrumb .= '<span itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><meta itemprop="position" content="' . esc_html( $position ) . '"><a href="' . esc_url( get_permalink( $ancestor ) ) . '" itemprop="item"><span itemprop="name">' . esc_html( get_the_title( $ancestor ) ) . '</span></a></span>';
```

Comment:
Do we want to `esc_attr` all those instead of `esc_html`?

Reply:
Good idea. Blanked a bit. In theory, it probably isn't needed since $position is set all in the code, but better to future-proof it since the spec allows the position field to be text.

b0ec59a